File: atividade-12/atividade.go
package main

import "fmt"

func somar(a int) int {
	return a
}

func somarDois(a int, b int) int {
	return a + b
}

func somarTres(a int, b int, c int) int {
	return a + b + c
}

func main() {
	fmt.Println(somar(1))           
	fmt.Println(somarDois(1, 2))    
	fmt.Println(somarTres(1, 2, 3)) 
}

File: atividade-12/atividade.py
class Calculadora:
    def somar(self, a, b=0, c=0):
        return a + b + c

calc = Calculadora()
print(calc.somar(1))     
print(calc.somar(1, 2))   
print(calc.somar(1, 2, 3))  